opentelemetry-api/tests/resources/test_init.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+from opentelemetry.resources import Resource
+
+
+class TestResources(unittest.TestCase):
+    @staticmethod
```

Comment:
In OC we generally made unit tests instance methods, even if only to use `self.assertSomething`-style asserts.

Reply:
the linter failed the test due to my lack of using the self. I can use the self.asserts to avoid that.

On an aside I prefer pytest, but I'll probably file a separate issue to discuss that.

Reply:
You and @Oberon00 both. We talked about this earlier (see https://github.com/open-telemetry/opentelemetry-python/issues/28) but decided to start with bare unit tests and consider moving to pytest once we've got some of the SDK written and tested.

Reply:
switched to self.assert calls, satisfying the linter.